resources/skins/nudus-base/html_style_base.scss

```diff
@@ -191,7 +181,8 @@ pre {
     background-color: $ccodeblock;
     overflow-x: auto;
     padding: 7px 13px;
-    // For cases of <pre style='white-space:pre-wrap;'>
+    tab-size: 4;
```

Comment:
Personally, I like just 2 tabs/spaces, just a comment.

Reply:
Ah, of course you do.